Sources/docc/DocCDocumentation.docc/linking-to-symbols-and-other-content.md

```diff
@@ -103,65 +150,146 @@ The following example shows a symbol link to the `red` static property:
 ``Color/red-type.property``
 ```
 
-DocC supports the following symbol types for use in symbol links:
+DocC supports the following symbol types as disambiguation in symbol links:
 
 | Symbol type       | Suffix            |
 |-------------------|-------------------|
 | Enumeration       | `-enum`           |
 | Enumeration case  | `-enum.case`      |
 | Protocol          | `-protocol`       |
-| Operator          | `-func.op`        |
 | Typealias         | `-typealias`      |
-| Function          | `-func`           |
+| Associated Type   | `-associatedtype` |
 | Structure         | `-struct`         |
 | Class             | `-class`          |
+| Function          | `-func`           |
+| Operator          | `-func.op`        |
+| Property          | `-property`       |
 | Type property     | `-type.property`  |
+| Method            | `-method`         |
 | Type method       | `-type.method`    |
+| Subscript         | `-subscript`      |
 | Type subscript    | `-type.subscript` |
-| Property          | `-property`       |
 | Initializer       | `-init`           |
 | Deinitializer     | `-deinit`         |
-| Method            | `-method`         |
-| Subscript         | `-subscript`      |
+| Global variable   | `-var`            |
 | Instance variable | `-ivar`           |
 | Macro             | `-macro`          |
 | Module            | `-module`         |
+| Namespace         | `-namespace`      |
+| HTTP Request      | `-httpRequest`    |
+| HTTP Parameter    | `-httpParameter`  |
+| HTTP Response     | `-httpResponse`   |
+| HTTPBody          | `-httpBody`       |
+| Dictionary        | `-dictionary`     |
+| Dictionary Key    | `-dictionaryKey`  |
 
-Symbol type suffixes can include a source language identifier prefix — for 
-example,  `-swift.enum` instead of `-enum`. However, the language 
-identifier doesn't disambiguate the link.
+You can discover these symbol type suffixes from DocC's warnings about ambiguous symbol links.
+DocC suggests one disambiguation for each of the symbols that match the ambiguous symbol path.
 
-Symbol paths are case-sensitive, meaning that symbols with the same name in
-different text casing don't need disambiguation. 
+Symbol type suffixes can include a source language identifier prefix---for example, `-swift.enum` instead of `-enum`. 
+However, the language identifier doesn't disambiguate the link.
```

Comment:
If it doesn't disambiguate the link, why would someone use this suffix?

Reply:
In the first few releases of DocC (before Xcode 14.3) DocC required the language identifier prefix for unrelated and uninteresting implementation detail reasons, even though the that information wasn't useful for disambiguation purposes. Links created during those releases continue to work but can be shortened.